.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "openapi"]
+	path = internal/api/contract/openapi
+	url = ../openapi
```

Comment:
Using a relative URL here is a trick that allows cloning the submodule using either `https` or `ssh`, depending on how you cloned the `influx-cli` repo.